src/main/resources/jenkins/plugins/git/Messages.properties

```diff
@@ -26,3 +26,5 @@ GitStep.git=Git
 within.Repository=Within Repository
 additional=Additional
 GitUsernamePasswordBinding.DisplayName=Git Username and Password
+
+git.fips.url.notsecured=Invalid configuration will not fetch any remote. FIPS requires to use git URL with TLS.
```

Comment:
Proposed alternate phrasing.  Optional, since this is just my phrasing preference.
```suggestion
git.fips.url.notsecured=Invalid configuration will not fetch any remote. FIPS requires a secure channel for git fetch with credentials.
```